modules/lx-autoscale/watchmaker-lx-autoscale.cfn.json

```diff
@@ -519,9 +506,8 @@
             "Type": "String"
         },
         "WatchmakerConfig": {
-            "AllowedPattern": "^$|^http[s]?://.*$",
```

Comment:
I'd like to keep the `AllowedPattern`. It's a regex, so we should be able to include several options.

Maybe this: `"^$|^(http[s]?|file|s3)://.*$"`. That would not allow paths like `/` or `C:`, but those can be passed as `file://...` instead.

https://regex101.com/r/aSi2LD/1

Reply:
I agree.  I'll make the recommended changes.

Reply:
Just realized you may be referring to only to the Linux templates?  I made the changes to the Windows ones as well.  I did test with "file://%TEP%/config.yml" on a Windows instance and it still worked properly.

Reply:
All templates. The new uri methods support `file://` on windows as well. Examples in the pr that merged the new approach: https://github.com/plus3it/watchmaker/pull/507